third_party/kourier-latest/kourier.yaml

```diff
@@ -299,7 +299,7 @@ spec:
         app: net-kourier-controller
     spec:
       containers:
-        - image: gcr.io/knative-nightly/knative.dev/net-kourier/cmd/kourier@sha256:5216c4465db7932d77754059d0016e6b6f8a39b4189cd9234c821c1dfe8342cc
+        - image: gcr.io/gcp-compute-engine-223401/kourier-b74c3918b7eee585f87df62ccd297dc8@sha256:bf80c24b17f79f306a39853a7e9a96b532fec7e752346edcb4e45bdda71d6c4e
```

Comment:
This is updated automatically, we could fetch deps update and then rebase but ok.